src/web/core/src/main/java/org/geoserver/web/ServiceDescriptionProvider.java

```diff
@@ -120,22 +131,37 @@ protected ServiceDescription description(
                 layerInfo != null ? layerInfo.getName() : null);
     }
 
+    /**
+     * gets the name of the "version" parameter for the service. This will usually be "version", but
+     * some (i.e. WCS 2+) it will be "acceptversions". To overrided by subclasses.
+     *
+     * @param service
+     * @return
```

Comment:
```suggestion
     * Gets the name of the {@code version} parameter for the service. This will usually be {@code version}, but
     * some (i.e. WCS 2+) it will be {@code acceptversions}. To overrided by subclasses.
     *
     * @param service
     * @return version parameter of service, example {@code version} or {@code acceptversions}
```